run-container/README.md

```diff
@@ -0,0 +1,116 @@
+# How to run the container for the Single cell RNA Seq Snap workflow (ScRNASeqSnap)
+
+We have generated Dockerfile and Definition file that contain all tools, packages, and dependencies necessary to run the code and analyses modules in the `sc-rna-seq-snap` repository. These are customized for `Rstudio/R v4.4.0` and `Seurat v4.4.0`.
+
+
+## To use the container in an R interactive session on HPC:
+
+From an interactive node on HPC, the user can open their session. Please modify memory and resources as needed for the analysis module to run.
+```
+bsub -P hpcf_interactive -J hpcf_interactive -n 1 -q standard -R "rusage[mem=4G]" -Is "bash"
+```
+
+## Load specific version of Singularity
+
+Please note that a version of Singularity is installed by default on all the cluster nodes at St Jude HPC. 
+Otherwise the user needs to ensure and load Singularity module by running:
+```
+module load singularity/4.1.1
+```
+
+
+1. Clone the repository
+```
+git clone https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap.git
+```
+
+2. Pull the singularity container
+```
+singularity pull docker://wabuala/rstudio_4.4.0_seurat_4.4.0:latest
```

Comment:
This command failed to complete running in an the interactive session with the command as is above (aka with 4GB memory). I tested with 8GB and that also failed to complete. Essentially it would get to the step of creating the SIF file, but then would automatically exit the interactive node (after a bit of time) without the sif file being created. I intended to test a higher memory, but unintentionally ran the singularity pull command from the login node. The command did complete on the login node, so now because I have the SIF file available/cached already, I'm not able to try testing any higher memories on the interactive node properly.

Reply:
Please clean the cache, remove all files, and try again with the new resources as updated here 515b9d1f29838204dddb7f06e6b28437cc0171b3.